features/bootstrap/FeatureContext.php

```diff
@@ -191,6 +192,30 @@ public function iSetBehatParamsEnvironmentVariable(PyStringNode $value)
         $this->env = array('BEHAT_PARAMS' => (string) $value);
     }
 
+    /**
+     * @When I set the working directory to :dir
+     */
+    public function iSetTheWorkingDirectoryTo($dir): void
+    {
+        $this->workingDir = realpath($dir);
```

Comment:
instead of resolving the directory based on the current CWD and setting it to `false` in case the directory does not exist, I would suggest making the code more robust:

- treat it explicitly as a a relative path based on `dirname(__DIR, 2)` (or maybe omit the `tests/Fixtures` part of the path by moving that to the base path, to enforce that scenarios uses those fixtures and not something else)
- if we don't support using `..` in that relative path, I don't think we need to use `realpath` (we won't have symlinks in our fixtures anyway, as those don't play well with a git repo aimed at being usable cross-platform)
- add a `is_dir` check to report a proper error if the provided directory does not exist instead of getting weird errors later in the scenario.